packages/next/build/index.ts

```diff
@@ -56,14 +56,14 @@ export default async function build(dir: string, conf = null): Promise<void> {
   const pages = createPagesMapping(pagePaths, config.pageExtensions)
   const entrypoints = createEntrypoints(pages, config.target, buildId, config)
   const configs = [
```

Comment:
This used to be the below line, but if you prefer this it works 👌
```js
const configs: any = await Promise.all([
```

Reply:
I'm aware 😄